settings.gradle

```diff
@@ -11,7 +11,9 @@ pluginManagement {
   }
 }
 
-rootProject.name = 'ktlint'
+rootProject.name = 'ktlint-root'
```

Comment:
Rename the rootProject to solve potential name conflict with subproject `ktlint`.